### ISSUE TITLE
[question][macOS] The default Arduino data directory location on macOS

### ISSUE THREAD

**Issue by kittaakos** [arduino/arduino-cli#290]
### Current behavior

I am not sure if this is a bug, but the CLI has a different default data directory location than the one the Arduino editor uses.

For the CLI, it is `ardunio15`:
https://github.com/arduino/arduino-cli/blob/296c87275f0c746c2b66768fcc584813963453cd/configs/directories.go#L47

In case of the editor, it is `Arduino15`:
https://github.com/arduino/Arduino/blob/fb9ce0ade323f6f36793f29d615db1272d184ba4/arduino-core/src/processing/app/macosx/Platform.java#L80

### Environment

- CLI version (output of `arduino-cli version`):
- Go version (if building from sources):
`go version go1.12.6 darwin/amd64`
- OS version:
`macOS 10.14.4 (18E226)`.

**Comment by gvarisco** [comment]
I noticed this yesterday on _macOS 10.14.6_ as well. 

tl;dr: @kittaakos is right, the name of the folder is not consistent (**arduino15** vs **Arduino15**). But given starting w/ macOS 10.12 High Sierra, the default FS changed to case-insensitive APFS, no bad things are happening. 

But to be on the safe side, I do suggest we resolve all case-sensitivity naming conflicts (that is, IDE and CLI) to be consistent. Keeping in mind macOS' capitalization "best practice".
